examples/jupyter/README.md

```diff
@@ -72,24 +83,24 @@ the Jupyter notebook for local interaction as follows.
 ### Setup
```

Comment:
You could mention that you need to be in `examples/jupyter` for this.

Reply:
This is the README of `examples/jupyter`.